aait/backend/group-2A/WebApi/Controllers/CommentController.cs

```diff
@@ -0,0 +1,68 @@
+﻿using Application.DTO.CommentDTO;
+using Application.Features.Comment.Requests.Commands;
+using Application.Features.Comment.Requests.Queries;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+
+namespace WebApi.Controllers;
+
+[ApiController]
+[Route("api/[controller]")]
+public class CommentController : ControllerBase
+{
+    private readonly IMediator _mediator;
+
+    public CommentController(IMediator mediator)
+    {
+        _mediator = mediator;
+    }
+    
+    // POST / Comment   
+    [HttpPost]
+    public async Task<ActionResult<int>> AddComment([FromBody] CreateCommentDto commentDto)
+    {
+        var command = new CreateCommentCommand{ CommentDto = commentDto };
+        var commentId = await _mediator.Send(command);
+        return Ok(commentId);
```

Comment:
instead of returning Ok with just a number id when the object created , customize it to let the user know what was just happened.